app/models/saved_claim/veteran_readiness_employment_claim.rb

```diff
@@ -74,6 +74,10 @@ class SavedClaim::VeteranReadinessEmploymentClaim < SavedClaim
     '000' => 'VRE.VBAPIT@va.gov'
   }.freeze
 
+  before_create do
+    self.form_id = parsed_form['useEva'].present? ? self.class::FORM.upcase : '28-1900-V2'
```

Comment:
Looks like form is `28-1900` so there's nothing to upcase. And `FORMV2 = '28-1900_V2'`. Why does the constant use an underscore but we want a dash here? Seems like we use dashes in `filler.rb`. Would it be better to use dashes everywhere?